### DISCUSSION TITLE
Contact Members

### DISCUSSION THREAD

**Discussion by csgdeskgithub** [community/community#81007] [New to GitHub]
How can we contact outside members of our GitHub organisation of the pending requirement of 2FA?

**Comment by queenofcorgis** [comment]
👋🏼 Thanks for asking this @csgdeskgithub , 

To make an announcement we'd recommend using an Issue or a Discussion. If they do allow you to view their email, that is an option as well.  

If anyone else has best practices or experiences with outside collaborators and 2FA they'd like to share, we'd love to hear them!

**Reply by csgdeskgithub** [reply]
Thank you for your reply.
Can you provide any additional details on how to use the Issue or Discussion feature to communicate with collaborators, we do not any visibility of their email address.

**Reply by queenofcorgis** [reply]
Happy new year! I would recommend using the feature that you already use with collaborators - Issues may be better as you can both pin them and assign them to certain teams which may be the best way to get it in front of your folks.

**Reply by csgdeskgithub** [reply]
Happy new year!  Would I need to navigate to each repo in our Org and create a separate issue in each?

**Reply by queenofcorgis** [reply]
Hi there @csgdeskgithub - if you were to use Issues you would have to create a new one in each repo or at least in the repos where you wanted to inform folks.

**Comment by akashdip2001** [comment]
If you want to notify outside members of your GitHub organization about the pending requirement of Two-Factor Authentication (2FA), you can follow these steps:

1. **Using Email:**
   - If you have the email addresses of the external members, you can send an email notifying them of the upcoming 2FA requirement.
   - Clearly explain the importance of enabling 2FA for security reasons and provide any necessary instructions or links to relevant GitHub documentation.

2. **Organization-Wide Message:**
   - If your GitHub organization has a "Team" or "Discussion" feature, you can use it to post an organization-wide message.
   - Create a post informing members about the upcoming 2FA requirement, and include instructions or links for setting up 2FA.

3. **Repository README or Wiki:**
   - If your organization has a central repository that external members have access to, you can update the README or Wiki with information about the 2FA requirement.
   - This way, whenever they access the repository, they will see the notice.

4. **GitHub Organization Announcements:**
   - GitHub might have introduced new features or settings after my last update in January 2022. Check the organization settings or announcements section on GitHub to see if there are built-in tools for communicating with organization members.

Remember to communicate the 2FA requirement well in advance and provide clear instructions on how external members can enable 2FA on their GitHub accounts. If possible, offer assistance or support for any questions they might have during the process.

## Profile
![Profile Image](https://thumbnails.odycdn.com/optimize/s:200:0/quality:95/plain/https://thumbs.odycdn.com/bf58cf4d7ec52c62f02c52e16f150a5b.webp)
**AKASHDIP MAHAPATRA**
"everything is physics bro"  
E=mc²

|[Subscribe on YouTube](https://www.youtube.com/channel/UCxvmp634YDc41xCWOdvWqoQ?sub_confirmation=1) |
|----  |
